bears/general/LicenseCheckBear.py

```diff
@@ -0,0 +1,37 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.DistributionRequirement import (
+    DistributionRequirement)
+
+
+@linter(executable='licensecheck',
+        output_format='regex',
+        output_regex=r'.*: .*UNKNOWN',
```

Comment:
Can you be a bit more precise than `.*` -- it is probably going to be OK, but better to be safer to avoid strange breakages later on.